types/artifacts.d.ts

```diff
@@ -69,6 +69,8 @@ declare global {
       RuntimeExceptions: Crdp.Runtime.ExceptionThrownEvent[];
       /** Information on all script elements in the page. Also contains the content of all requested scripts and the networkRecord requestId that contained their content. Note, HTML documents will have one entry per script tag, all with the same requestId. */
       ScriptElements: Array<Artifacts.ScriptElement>;
+      /** Source maps of scripts executed in the page. */
+      SourceMaps: Array<Artifacts.SourceMap>;
```

Comment:
I don't think we should commit to no breaking changes on these yet, so these should be in `GathererArtifacts`, not `PublicGathererArtifacts`

Reply:
Agreed, wasn't aware I placed it in public.